compile-release-tools

```diff
@@ -18,6 +18,8 @@ set -e
 set -u
 set -o pipefail
 
+GOPATH=$(go env GOPATH)
```

Comment:
i would set that in the line above of 37

```
      export RELEASE_TOOL_BIN="${GOPATH}/bin"
```

Reply:
Done!